docs/source/markdown/podman-events.1.md

```diff
@@ -104,20 +104,22 @@ In the case where an ID is used, the ID may be in its full or shortened form.  T
 
 Format the output to JSON Lines or using the given Go template.
 
-| **Placeholder**       | **Description**                               |
-|-----------------------|-----------------------------------------------|
-| .Attributes           | created_at, _by, labels, and more (map[])     |
-| .ContainerExitCode    | Exit code (int)                               |
-| .ContainerInspectData | Payload of the container's inspect            |
-| .HealthStatus         | Health Status (string)                        |
-| .ID                   | Container ID (full 64-bit SHA)                |
-| .Image                | Name of image being run (string)              |
-| .Name                 | Container name (string)                       |
-| .Network              | Name of network being used (string)           |
-| .PodID                | ID of pod associated with container, if any   |
-| .Status               | Event status (e.g., create, start, died, ...) |
-| .Time                 | Event timestamp (string)                      |
-| .Type                 | Event type (e.g., image, container, pod, ...) |
+| **Placeholder**         | **Description**                               |
+|-------------------------|-----------------------------------------------|
+| .Attributes ...         | created_at, _by, labels, and more (map[])     |
+| .ContainerExitCode      | Exit code (int)                               |
+| .ContainerInspectData   | Payload of the container's inspect            |
+| .HealthStatus           | Health Status (string)                        |
+| .ID                     | Container ID (full 64-bit SHA)                |
+| .Image                  | Name of image being run (string)              |
+| .Name                   | Container name (string)                       |
+| .Network                | Name of network being used (string)           |
+| .PodID                  | ID of pod associated with container, if any   |
+| .Recycle *path* *bool*  | [unimplemented]                               |
```

Comment:
this look like a bogus function that is never called or used.

Mind removing it with
```diff
diff --git a/libpod/events/events.go b/libpod/events/events.go
index bcda07e06..18f531469 100644
--- a/libpod/events/events.go
+++ b/libpod/events/events.go
@@ -54,13 +54,6 @@ func NewEvent(status Status) Event {
        }
 }
 
-// Recycle checks if the event log has reach a limit and if so
-// renames the current log and starts a new one.  The remove bool
-// indicates the old log file should be deleted.
-func (e *Event) Recycle(path string, remove bool) error {
-       return errors.New("not implemented")
-}
-
 // ToJSONString returns the event as a json'ified string
 func (e *Event) ToJSONString() (string, error) {
        b, err := json.Marshal(e)
```

Otherwise I can submit a PR to remove it because there should really be no point in documenting broken stuff that will never help any user.

Reply:
Done. Required small tweak to regression test. I updated the commit message.

`.Recycle()` dates back to #2527 in 2019. Tagging @baude - that was your code, it looks like a placeholder that got orphaned. If you see a strong reason to leave that, please let me know.

Reply:
Brent has enough on his hands this week. I'm gonna say if there's ever a need for `.Recycle()` it can be added (and actually implemented) then. @containers/podman-maintainers PTAL, I consider this ready to merge.